src/Router.php

```diff
@@ -247,15 +246,13 @@ public static function catch(string $toCatch, string $controller, ?string $uri =
 
     public static function makeCatchable(string $toCatch): void
     {
-        if (isset(self::$catchers[$toCatch])) {
-            return;
-        }
+        if (!isset(self::$catchers[$toCatch])) {
+            if (!is_subclass_of($toCatch, HttpException::class)) {
```

Comment:
Thanks, I hate it

Reply:
It was the only way to get full coverage. Not sure why it was complaining 🫠

Reply:
Use an interface and `instanceof`, this is jank

Reply:
$toCatch is a string, instanceof won't work

Reply:
Up to you I suppose, I don't see why you make an effort to filter out non-`HttpException` in the first place - I don't see how it could hurt to just add this into the array regardless. Worst case scenario it wastes some ram by storing it.